requirements.txt

```diff
@@ -0,0 +1,3 @@
+black
```

Comment:
Did you mean to name this file `requirements-dev.txt`? You have reference that above. It is good to make it clear that this is not required for running Python code.